app/code/Magento/Config/App/Config/Source/RuntimeConfigSource.php

```diff
@@ -60,7 +60,7 @@ public function __construct(
     public function get($path = '')
     {
         $data = new DataObject($this->loadConfig());
-        return $data->getData($path) ?: [];
+        return $data->getData($path);
```

Comment:
For better backward compatibility I'd do the following:
```suggestion
        return $data->hasData($path) ? $data->getData($path) : [];
```
Also please correct the `return` annotation to `@return mixed`